about/github-instructions/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+status: publish
+published: true
+title: 'Instructions on using GitHub to Contribute to gis.utah.gov'
+author:
+  display_name: Jessie Pechmann
+  email: jpechmann@utah.gov
+date: 2016-04-19 16:11:21
+categories: Featured
+tags: how-to
+---
+
+**Method 1: Edit an existing page from website**
+
+1. Create a free [GitHub account](https://github.com/join), if you do not already have one.
+* Click ‘ Edit this page on GitHub ’ in the bottom right of any webpage on gis.utah.gov.  ![Edit this page on GitHub]({{ "/images/editgithub.png" | prepend: site.baseurl }})  
```

Comment:
looks like i had to switch these to 1.'s to work. the * trick is only for github flavored markdown.